app/components/layer-template-select/component.js

```diff
@@ -5,6 +5,7 @@ import TimeMachine from 'ember-time-machine';
 import ENV from '../../config/environment';
 
 export default Ember.Component.extend({
+    classNames: ['d-inline'],
```

Comment:
not the most descriptive name for a class

Reply:
@cameronblandford its part of bootstrap 4 but since we are using bootstrap 3 i made my own 
https://v4-alpha.getbootstrap.com/utilities/display-property/